### ISSUE TITLE
Time data format error with nonlinloc?

### ISSUE THREAD

**Issue by angelcalayag** [megies/obspyck#13]
I'm using obspyck on Ubuntu virtual machine. When performing "do nlloc", the error below appears.  I checked the temp output file "last.hyp" and looked OK so it seemed to do the location correctly. I compared this file with another obspyck nlloc output that ran on Mac and are very similar. What could be causing the creation_time formatting error in python?

```
/tmp/obspyck-vIOENt/nlloc/nlloc*
Phases for NLLoc:
RJOB   ?    ?    ? P      ? 20130612 1310 41.1300 GAU  5.00e-02 -1.00e+00 -1.00e+00 -1.00e+00
RJOB   ?    ?    ? S      ? 20130612 1310 41.5950 GAU  3.50e-02 -1.00e+00 -1.00e+00 -1.00e+00
RTBE   ?    ?    ? P      ? 20130612 1310 40.8050 GAU  3.00e-02 -1.00e+00 -1.00e+00 -1.00e+00
RTBE   ?    ?    ? S      ? 20130612 1310 41.0700 GAU  3.50e-02 -1.00e+00 -1.00e+00 -1.00e+00
RTSA   ?    ?    ? P      ? 20130612 1310 41.1400 GAU  2.50e-02 -1.00e+00 -1.00e+00 -1.00e+00
RTSA   ?    ?    ? S      ? 20130612 1310 41.4850 GAU  2.50e-02 -1.00e+00 -1.00e+00 -1.00e+00


--> NLLoc finished
NLLOC "./nlloc.20130612.131040.grid0" "LOCATED" "Location completed."
SIGNATURE "Megies LMU Munich   NLLoc:v5.00.0 29May2015 17h44m47"
COMMENT "NonLinLoc OctTree Location"
GRID  179 179 87  4553.900000 5281.100000 -1.700000  0.100000 0.100000 0.100000 PROB_DENSITY
SEARCH OCTREE nInitial 300 nEvaluated 50012 smallestNodeSide 0.027813/0.027813/0.044792 oct_tree_integral 4.472157e-01 scatter_volume 4.472157e-01
HYPOCENTER  x 4563.008594 y 5290.069531 z 1.413021  OT 40.524281  ix -1 iy -1 iz -1
GEOGRAPHIC  OT 2013 06 12  13 10 40.524281  Lat 5290.069531 Long 4563.008594 Depth 1.413021
QUALITY  Pmax 2.574e-01 MFmin 0.672584 MFmax 8512.712518 RMS 0.025135 Nphs 6 Gap 246 Dist 1.109662 Mamp -9.90 0 Mdur -9.90 0
VPVSRATIO  VpVsRatio 4.500e+00  Npair 3  Diff 2.000e-01
STATISTICS  ExpectX 4.5631e+03 Y 5.2900e+03 Z 8.3035e-01  CovXX 2.33e-01 XY -2.13e-01 XZ -9.83e-02 YY 1.84e-01 YZ 8.39e-02 ZZ 6.91e-01 EllAz1  41.8 Dip1  0.2 Len1  1.47e-01 Az2  131.9 Dip2  22.0 Len2  1.14e+00 Len3  1.62e+00
STAT_GEOG  ExpectLat 5289.952731 Long 4563.146374 Depth 0.830345
TRANSFORM  NONE
FOCALMECH  Hyp  5290.069531 4563.008594 1.413021 Mech  0.0 0.0 0.0 mf  0.00 nObs 0
PHASE ID Ins Cmp On Pha  FM Date     HrMn   Sec     Err  ErrMag    Coda      Amp       Per  >   TTpred    Res       Weight    StaLoc(X  Y         Z)        SDist    SAzim  RAz  RDip RQual    Tcorr 
RTBE   ?    ?    ? P      ? 20130612 1310   40.8050 GAU  3.00e-02 -1.00e+00 -1.00e+00 -1.00e+00 >    0.3107   -0.0300    1.0743 4561.9031 5289.9737    0.0000    1.1097 265.05 263.8 142.7  9     0.0000
RTBE   ?    ?    ? S      ? 20130612 1310   41.0700 GAU  3.50e-02 -1.00e+00 -1.00e+00 -1.00e+00 >    0.5219    0.0238    0.8595 4561.9031 5289.9737    0.0000    1.1097 265.05 263.8 142.7  9     0.0000
RTSA   ?    ?    ? P      ? 20130612 1310   41.1400 GAU  2.50e-02 -1.00e+00 -1.00e+00 -1.00e+00 >    0.5831    0.0326    1.3626 4563.2512 5292.4571    0.0000    2.3998   5.80   8.1 123.9  9     0.0000
RTSA   ?    ?    ? S      ? 20130612 1310   41.4850 GAU  2.50e-02 -1.00e+00 -1.00e+00 -1.00e+00 >    0.9796   -0.0189    1.3626 4563.2512 5292.4571    0.0000    2.3998   5.80   8.1 123.9  9     0.0000
RJOB   ?    ?    ? P      ? 20130612 1310   41.1300 GAU  5.00e-02 -1.00e+00 -1.00e+00 -1.00e+00 >    0.6331   -0.0274    0.4816 4559.7848 5289.0697    0.0000    3.3753 252.77 252.9 112.5  9     0.0000
RJOB   ?    ?    ? S      ? 20130612 1310   41.5950 GAU  3.50e-02 -1.00e+00 -1.00e+00 -1.00e+00 >    1.0636    0.0071    0.8595 4559.7848 5289.0697    0.0000    3.3753 252.77 252.9 112.5  9     0.0000
END_PHASE
END_NLLOC


Traceback (most recent call last):
  File "./obspyck.py", line 416, in on_qToolButton_doNlloc_clicked
    self.loadNLLocOutput()
  File "./obspyck.py", line 2023, in loadNLLocOutput
    creation_time = UTCDateTime().strptime(date + time, str("%d%b%Y%Hh%Mm%S"))
  File "/usr/local/lib/python2.7/dist-packages/obspy/core/utcdatetime.py", line 1152, in strptime
    return UTCDateTime(datetime.datetime.strptime(date_string, format))
  File "/usr/lib/python2.7/_strptime.py", line 325, in _strptime
    (data_string, format))
ValueError: time data '29May201517h44m47' does not match format '%d%b%Y%Hh%Mm%S'
```

**Comment by megies** [comment]
Seems you're running a default locale which is not "en_US.UTF-8" and thus not able to make sense of "May" in the timestring. I'll try to fix it in a commit, let me know if it still doesn't work.

**Comment by megies** [comment]
closed with 65e7d3db872c6ea1ca3e3
